handbook/product/rollout_process.md

```diff
@@ -0,0 +1,38 @@
+# Rollout Process
+
+The rollout process consists of *Pre-Launch* phase and *Deployment* phase. We can push changes to Sourcegraph.com when we
+want as we own the deployment process. For enterprise instances of Sourcegraph.com, we push a new release monthly and our
+clients decide to upgrade to the latest release when they want.
+
+### Sourcegraph.com
+
+#### Pre-Launch
+1. User Testing 
+- Hallway test with internal users.
+- Open the feature to internal users by turning it on for the Sourcegraph organization. Make sure it's opened with enough
+time left before planned launch date to receive and address all the feedback received. 
+2. Design 
+- Test the feature for both themes, Light and Dark. Check for icon consistency across themes.
+* Test on small, medium, large and extra-large screen sizes.
+  * Small screens are important to consider for a good experience when the window is resized or in side-by-side mode.
+* Code examples in design
+  * Font should be "code font" or monospace.
+  * Easy to copy and paste (no fancy quotes).
```

Comment:
May want to add that these are only applicable for UI-facing changes (or alternatively, that this document lays out the process for rolling out UI-facing launches)

Reply:
I don't know what rolling out backend changes would look like. For now, followed your above comment and added a line to skip if not applicable.